Containerfile.nvidia-open

```diff
@@ -0,0 +1,63 @@
+###
+### Containerfile.nvidia - used to build ONLY NVIDIA kmods
```

Comment:
I think we can just symlink this instead of it's going to just be an exact copy of the Nvidia containerfile.

Reply:
Right now it's separate but it could be an if in the containerfile